src/content/docs/blog/biome-v2-0-beta.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Biome v2.0 beta
+description: A major milestone in the making
+summary: |
+  Biome 2.0 will be packed with major features and many smaller fixes, rules, and other improvements.
+  There's a lot to unpack, and we request the community's help testing this beta, so the final
+  release can be as smooth as possible.
+authors:
+  - arendjr
+  - team
+date: 2025-03-31
+cover:
+  light: "@/assets/blog/roadmap-2024/banner-light.png"
+  dark: "@/assets/blog/roadmap-2024/banner-dark.png"
+  alt: The brand of the project. It says "Biome, toolchain of the web"
+socialImage: "@/assets/social-logo.png"
+---
+
+After hard work from our team, Biome's long-awaited 2.0 release is nearing completion. It will be packed with many large features, so we would like your help testing it with a public beta!
+
+If you would like to try it out, you can update Biome and migrate your configuration using the following commands:
+
+```shell
+npm install --save-dev --save-exact @biomejs/biome@beta
+npx @biomejs/biome migrate
```

Comment:
```suggestion
npx @biomejs/biome@beta migrate
```

`npx` downloads packages straight from the registry if I remember correctly

Reply:
Only if the package is not installed, I thought, which is the confusing (and rather insecure) thing about `npx`: it is used for both local and remote command invocations. You make one typo trying to execute a local command, and you'll be downloading and executing code off the internet :sweat: